### PR TITLE
Update python packages pika

### PR DESCRIPTION
**Pull request by lsix**
###### Things done:
- [X] Tested using sandboxing (`nix-build --option build-use-chroot true` or [nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lancelotsix** [comment]
The build fails because of derivations that should be rebuilt but also depend on `sqlalchemy7` (`virtmanager` and `buildbot`). `salut_a_toi` also seems to be broken at the moment.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -15579,16 +15579,25 @@ in modules // {
...
-      url = https://pypi.python.org/packages/source/p/pika/pika-0.9.12.tar.gz;
-      sha256 = "670787ee6ade47cadd1ec8220876b9b7ae4df7bc4b9dd1d808261a6b47e9ce5d";
+      url = "https://pypi.python.org/packages/source/p/pika/${name}.tar.gz";
+      sha256 = "0nb4h08di432lv7dy2v9kpwgk0w92f24sqc2hw2s9vwr5b8v8xvj";
     };
-    buildInputs = with self; [ nose mock pyyaml ];
 
-    propagatedBuildInputs = with self; [ unittest2 ];
+    # Those use twisted which is only availalble for python2x
+    doCheck = !isPy3k;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: if that's the case, then this package won't be available for CPython 3.x anyway, so there is no need to add this.

Comment (lancelotsix) [inline pkgs/top-level/python-packages.nix]: `twisted` is only pulled when the interpreter is not cpython 3, and only used for testing. Therefore, `pika` is available for python3 (I have used and tested it) but no automatic test is run.

**Comment by lancelotsix** [comment]
Up. I have rebased and updated this PR. CI still fails because of broken buildbot dependency (sqlalchemy-0.7).

**Comment by FRidh** [comment]
Pushed to master.

**Comment by FRidh** [comment]
5c436c7a763357350b5297af0ad9ab613be46fb3 and b5e237bda66095eaa3c071ba58b1282ea4a45651

**Pull request closed without merging**